### DISCUSSION TITLE
Snippet helper on attribute with no match

### DISCUSSION THREAD

**Discussion by spencerkent** [algolia/instantsearch#5018] [Q&A] [answered]
My search has queryBy set on 3 attributes in my index, so my UI displays all three of these attributes in a hit. Two are short and I use the [highlight helper](https://www.algolia.com/doc/api-reference/widgets/highlight/js/), **the 3rd is very long** so I only display a snippet using the [snippet helper](https://www.algolia.com/doc/api-reference/widgets/snippet/js/#widget-param-attribute). 

### My problem
If the match comes from either of the first two attributes, that third attribute is displayed in full, no snippet. Of course it works fine when the hit is actually on the snippeted attribute--what is displayed is just the short snippet. 

### What I'm trying to do
I'd like to set a minimum amount of text to display for that third attribute (the one I've wrapped in the snippet helper) for when the hits are originating from the other attributes.

Can someone help me see how to do this?

**Comment by Haroenv** [comment, marked as answer]
Is it possible you're using a different search engine than Algolia? By default Algolia will always show the number of words you specify, even if there's no match in the word: https://codesandbox.io/s/confident-ives-ofjs4m?file=/src/app.js

**Reply by spencerkent** [reply]
Ah yes, I'm using Typesense, via their [instantsearch adapter](https://github.com/typesense/typesense-instantsearch-adapter).

**Reply by Haroenv** [reply]
That's a feature request for them then :)

**Reply by spencerkent** [reply]
Got it, I'll follow up with them, thank you.
